.github/workflows/add-author-issue_comment.yml

```diff
@@ -0,0 +1,24 @@
+  name: 'Add awaiting author on comment'
+
+  on:
+    issue_comment:
+      types: [created]
+
+  jobs:
+    label-pr:
+      name: Label PR
+      if: github.event.sender.login != github.event.issue.user.login && contains(github.event.issue.labels.*.name, 'review-requested') && !contains(github.event.issue.labels.*.name, 'Awaiting author contribution')
+      runs-on: ubuntu-latest
+      steps:
+        - name: author-commented
+          uses: actions/github-script@v6
+          with:
+            script: |
+              github.rest.teams.getMembershipForUserInOrg({
+                  org: 'conda-forge,
+                  team_slug: 'staged-recipes',
+                  username: github.event.sender.login
+              });
```

Comment:
This requires that the auction is authenticated with a user that has enough rights to see the 'staged-recipes' team. In practice, that's everybody in the conda-forge org.

Do we have a user that we can use here? 

Ref: https://github.com/actions/github-script#using-a-separate-github-token